README.md

```diff
@@ -63,7 +70,9 @@ Then, add `WoopraSDK` as a dependency for your target:
 targets: [
     .target(
         name: "YourAppName",
-        dependencies: ["WoopraSDK"]),
+        dependencies: [
+                .product(name: "WoopraSDK", package: "Woopra-iOS")]
```

Comment:
imo: is it correct? 🤔

Reply:
This should be the correct configuration.
Target initialization method: https://developer.apple.com/documentation/packagedescription/target/target(name:dependencies:path:exclude:sources:publicheaderspath:)
It requires `Target.Dependency` as the input type and `Product` conforms to it. (https://developer.apple.com/documentation/packagedescription/target/dependency/product(name:package:)-2nako)

Reply:
I tried again by creating a `Package.swift` in an empty folder by 
```
swift package init
```
Then,
1. Add the dependency according to the README.md
2. Run `swift build`

I found that the name in `dependencies: [.product(name: "WoopraSDK"... ` should be `Woopra`. 
Nice catch @cedricshih